frontend/src/component/menu/Header/NavigationMenu/NavigationMenu.tsx

```diff
@@ -46,19 +53,26 @@ export const NavigationMenu = ({
             open={Boolean(anchorEl)}
             style={style}
         >
-            {options.map(option => {
-                return (
+            {options.map((option, i) => (
+                <Fragment key={option.path}>
+                    <ConditionallyRender
+                        condition={Boolean(
```

Comment:
I'd extract this condition into intention revealing name/function. It's difficult to tell what options[i - 1] is.